samples/snippets/fsharp/query-expressions/query-expressions.fsproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You should be able to delete this file. There's a project file in the parent folder.